runtime/http.go

```diff
@@ -93,3 +111,11 @@ func (*httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
 	}
 }
+
+func flush(w http.ResponseWriter) {
+	if f, ok := w.(http.Flusher); ok {
+		// flush before unlocking so a panic failpoint won't
```

Comment:
This commend should be moved to `ServeHTTP` function as it directly references lock in there.

Reply:
sounds good & updated.